connector/launchers/runtime-embedded/build.gradle.kts

```diff
@@ -1,9 +1,23 @@
 plugins {
     `java-library`
     id("application")
+    id("jacoco")
     alias(libs.plugins.shadow)
 }
 
+tasks.test {
+    useJUnitPlatform() // Ensures JUnit 5 is used
+    finalizedBy(tasks.jacocoTestReport)
+}
+
+tasks.jacocoTestReport {
+    dependsOn(tasks.test)
+    reports {
+        xml.required.set(true)
+        html.required.set(true)
+    }
+}
+
```

Comment:
this is not needed anymore because launchers doesn't have tests on their own but get tested from the outside

Reply:
True. I'll remove